### ISSUE TITLE
Module04b fails when all regeno_bed files are empty

### ISSUE THREAD

**Issue by epiercehoffman** [broadinstitute/gatk-sv#77]
### Affected module(s) or script(s)
Module04b

### Description 
When all files in the `regeno_beds` input to Module04b are empty, Module04b.Genotype_2.GetRegenotype fails because it does not produce an output file, so Cromwell fails to de-localize the expected output file.

#### Expected behavior
Module04b should handle empty inputs rather than failing.

**Comment by epiercehoffman** [comment]
Pasting details here to keep track:

> For the PD cohort with 2 non-empty “median_missing_sorted.bed”, Module04b was finished with results.
> But for another cohort, LBD, with all empty “median_missing_sorted.bed”, the “master_regeno.bed” in /call-MergeList is empty and all /call-GetRegenotype scripts failed when scattering all cross 16 batches since no per batch .to_regeno.bed generated

> Use one batch as an example, the following script in call-GetRegenotype/ failed since it cannot generate output ".to_regeno.bed" because "master_regeno.bed" from call-MergeList is empty

```
svtk vcf2bed LBD_samples_2F2M_batch.2.depth.vcf.gz LBD_samples_2F2M_batch.2.bed
sample=$(fgrep -v "#" LBD_samples_2F2M_batch.2.bed|awk '{if($6!="" )print $6}' |head -n1|cut -d"," -f1) # a sample in batch is needed for regenotyping
while read chr start end variant s type; do
    printf "$chr\t$start\t$end\t$variant\t$sample\t$type\n">>LBD_samples_2F2M_batch.2.to_regeno.bed # modify this to suit intput for Rdtest
done <  /cromwell-execution/Module04b/6562d968-3734-4390-8f97-0ea097caaaa1/call-MergeList/master_regeno.bed
```

> Error message in GetRegenotype.log, it failed for attempting to copy .to_regeno.bed

```
Running command: sudo gsutil -q -m cp -L /var/log/google-genomics/out.log /mnt/local-disk/LBD_samples_2F2M_batch.2.to_regeno.bed gs://cromwell-execution/Module04b/6562d968-3734-4390-8f97-0ea097caaaa1/call-Genotype_2/shard-1/Regenotype/08570764-d41f-42bb-ae03-34d279798cb3/call-GetRegenotype/LBD_samples_2F2M_batch.2.to_regeno.bed
E: command failed: CommandException: No URLs matched: /mnt/local-disk/LBD_samples_2F2M_batch.2.to_regeno.bed
CommandException: 1 file/object could not be transferred.
```

**Comment by epiercehoffman** [comment]
Addressed in #86
